applications/FluidDynamicsApplication/python_scripts/adjoint_monolithic_solver.py

```diff
@@ -78,6 +139,8 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosCFD.AUX_ADJOINT_FLUID_VECTOR_1)
         self.main_model_part.AddNodalSolutionStepVariable(KratosCFD.ADJOINT_FLUID_SCALAR_1)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.VISCOSITY)
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.ADVPROJ)
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DIVPROJ)
```

Comment:
As far as I remember these are only required for the old VMS element. I'd put a comment to remind us removing them once we no longer support the old VMS element.

Reply:
added a comment.